charts/rancher-monitoring/v0.3.1/questions.yml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.6.1
```

Comment:
The 0.3.0 chart should have a max of 2.6.0

Reply:
`rancher_max_version: 2.6.0` is added to monitoring 0.3.0